packages/components/src/internal/components/forms/QueryFormInputs.tsx

```diff
@@ -243,30 +243,30 @@ export class QueryFormInputs extends React.Component<QueryFormInputsProps, State
                                 <React.Fragment key={i}>
                                     {this.renderLabelField(col)}
                                     <QuerySelect
+                                        addLabelAsterisk={showAsteriskSymbol}
                                         allowDisable={allowFieldDisable}
-                                        onToggleDisable={this.onToggleDisable}
-                                        initiallyDisabled={shouldDisableField}
                                         componentId={id}
+                                        containerPath={col.lookup.containerPath}
+                                        displayColumn={col.lookup.displayColumn}
                                         fireQSChangeOnInit={fireQSChangeOnInit}
+                                        formsy
+                                        initiallyDisabled={shouldDisableField}
                                         joinValues={joinValues}
                                         label={col.caption}
-                                        addLabelAsterisk={showAsteriskSymbol}
-                                        renderFieldLabel={renderFieldLabel}
-                                        loadOnChange
```

Comment:
similar question here...oh, this was likely a change to the default value for this prop

Reply:
Actually, this was a more subtle case where `loadOnChange` has been supplanted by `clearCacheOnChange` on `SelectInput` which now defaults to `true`.